features/dd-sdk-android-trace/transitiveDependencies

```diff
@@ -2,12 +2,18 @@ Dependencies List
 
 androidx.annotation:annotation:1.3.0                            :   30 Kb
 com.google.code.gson:gson:2.10.1                                :  276 Kb
+com.google.re2j:re2j:1.7                                        :  111 Kb
+com.squareup.okhttp3:okhttp:4.11.0                              :  768 Kb
+com.squareup.okio:okio-jvm:3.2.0                                :  337 Kb
```

Comment:
where it comes from? trace library shouldn't have its own transport, I was thinking we shouldn't have it after Moshi removal?

Reply:
I am pretty sure it is coming from something that is being used and not able to remove it but will double check it.